packages/ibm-products/src/components/ConditionBuilder/ConditionBuilderItem/ConditionBuilderItemOption/ConditionBuilderItemOption.js

```diff
@@ -83,7 +96,10 @@ export const ConditionBuilderItemOption = ({
   return (
     <>
       {allOptions && (
-        <div className={`${blockClass}__condition-builder-item-option`}>
+        <div
+          className={`${blockClass}__condition-builder-item-option`}
```

Comment:
this isn't new but it's something that just occurred to me: the block class is `export const blockClass = '${pkg.prefix}--condition-builder';` so here the class ends up being `condition-builder__condition-build-item-option`. i think the extra `condition-build` is a bit unnecessary. also i think you can make this slightly easier on yourself if you imported `blockClass` and a prefix and then instantiated a block class in each file so you don't have to repeat yourself as much. so something like
```
import { blockClass as prefix }
...
const blockClass = `${prefix}-item-option`
```
and this way you save on adding redundancy in class names and it becomes an easier pattern to use across the sub components.

Reply:
Hey,
Modified as advised